### ISSUE TITLE
Promotional-Assessment doesn't show correct response 

### ISSUE THREAD

**Issue by Tayzarcarr** [Tayzarcarr/d1095#1]
Promotional-Assessment doesn't show correct response 
![image](https://cloud.githubusercontent.com/assets/13252937/15612018/eedccd7c-2424-11e6-969c-a7c888bb68ab.png)

**Comment by billyfrost** [comment]
Yes, Storyline doesn't show the correct options - the text actually reads "all of the onscreen statements are correct", not "the correct statements are shown onscreen"
